### PR TITLE
Caffeine cache module

### PR DESCRIPTION
**Pull request by DamnClin**
Close https://github.com/jhipster/jhipster-lite/issues/236

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/server/springboot/cache/caffeine/infrastructure/primary/CaffeineCacheModuleConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  @Bean
+  JHipsterModuleResource caffeineCacheModule(CaffeineCacheApplicationService caffeineCaches) {
+    return JHipsterModuleResource
+      .builder()
+      .slug(JHLiteModuleSlug.CAFFEINE_CACHE)
+      .propertiesDefinition(JHipsterModulePropertiesDefinition.builder().addBasePackage().addIndentation().build())
+      .apiDoc("Spring Boot - Cache", "Add caffeine cache")
+      .organization(JHipsterModuleOrganization.builder().addDependency(JHLiteModuleSlug.SPRING_BOOT_CACHE).build())
+      .tags("server", "spring", "srpign-boot", "cache")
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/cache/caffeine/infrastructure/primary/CaffeineCacheModuleConfiguration.java:22]: Typo: srpign

File: src/main/resources/generator/server/springboot/cache/caffeine/caffeine.md.mustache [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+  @Bean
+  CacheManager usersCacheManager() {
+    CaffeineCacheManager manager = new CaffeineCacheManager("users");
+
+    manager.setCaffeine(Caffeine.newBuilder().expireAfterWrite(5, TimeUnit.MINUTES));
+
+    return manager;
+  }
+
```
Comment (pascalgrimaud) [inline src/main/resources/generator/server/springboot/cache/caffeine/caffeine.md.mustache:61]: Remove this empty line

**Review by pascalgrimaud — DISMISSED** [review]

File: tests-ci/generate.sh [hunk 3]
```diff
@@ -119,7 +119,9 @@ elif [[ $application == 'fullapp' ]]; then
   "java-archunit" \
   "github-codespaces" \
   "gitpod" \
-  "java-memoizers"
+  "java-memoizers" \
+  "spring-boot-cache" \
+  "caffeine-cache"
```
Comment (pascalgrimaud) [inline tests-ci/generate.sh:124]: Can we use caffeine with ehcache ? Maybe it's better to move this module to another build

Comment (DamnClin) [reply tests-ci/generate.sh:124]: Yes we can, no problem and totally logic as discussed on the ticket to rename ehcache feature to cache :⁠-⁠)

Comment (DamnClin) [reply tests-ci/generate.sh:124]: https://github.com/jhipster/jhipster-lite/issues/3598

**Pull request merged**